L/LibUnwind/LibUnwind@1.8.1/bundled/patches/libunwind-configure-static-lzma.patch

```diff
@@ -0,0 +1,20 @@
+--- configure.orig	2023-06-04 05:19:04
++++ configure	2023-06-07 08:35:11
+@@ -18117,7 +18117,7 @@
+   $as_echo_n "(cached) " >&6
+ else
+   ac_check_lib_save_LIBS=$LIBS
+-LIBS="-llzma  $LIBS"
++LIBS="-L/workspace/destdir/lib -l:liblzma.a $LIBS"
```

Comment:
Can't this be achieved by setting some vars, rather then patching the configure script?
```suggestion
+LIBS="-L${libdir} -l:liblzma.a $LIBS"
```